assets/js/blocks/attribute-filter/block.tsx

```diff
@@ -536,6 +536,21 @@ const AttributeFilterBlock = ( {
 
 	setWrapperVisibility( true );
 
+	const getIsApplyButtonDisabled = () => {
+		if ( termsLoading || countsLoading || checked.length === 0 ) {
```

Comment:
What about if I have one attribute checked, and I _uncheck_ this and go to click "Apply" assuming this would then have the same effect as the "Clear all" functionality.

Presumably based on this line, the "Apply" button will be disabled because `checked.length` will be `0`?

Reply:
@tjcafferkey thanks for pointing that out 🙏  As this condition was already there, I didn't touch it. But Looks like we don't need `checked.length === 0` condition anymore. 

**Why was this condition needed before?**
Because when the user initially load the page, no filter is selected therefore we needed this condition to keep the Apply disabled in this case.

**Why we don't need it anymore?**
Because now in this PR we add following code:
```jsx
const activeFilters = getActiveFilters( attributeObject );
if ( activeFilters.length === checked.length ) {
	return checked.every( ( value ) =>
		activeFilters.includes( value )
	);
}
```
This will return `true` when the user initially loads the page & Apply button will become disabled. Therefore, I believe it's safe to remove `checked.length === 0` condition.

I will remove this condition 👍

Reply:
@tjcafferkey FYI pushed the changes.